common/app/model/content/Atoms.scala

```diff
@@ -38,9 +39,30 @@ final case class MediaAtom(
   endSlatePath: Option[String],
   expired: Option[Boolean]
 ) extends Atom {
+
   def isoDuration: Option[String] = {
     duration.map(d => new Duration(Duration.standardSeconds(d)).toString)
   }
+
+  def formattedDuration: Option[String] = {
+    val formatter = new PeriodFormatterBuilder()
+      .appendHours
+      .appendSuffix(":")
+      .appendMinutes
+      .appendSuffix(":")
+      .printZeroAlways
+      .minimumPrintedDigits(2)
+      .appendSeconds
+      .toFormatter
+
+    duration.map(d => {
+      val duration =  new Duration(Duration.standardSeconds(d))
+      duration match {
+        case lessThanOneMinute if duration.isShorterThan(new Duration(Duration.standardMinutes(1))) => "0:" + formatter.print(duration.toPeriod())
```

Comment:
@TBonnin No, the `minimumPrintedDigits` is only useful to add padding to some value e.g. `3` becomes `03` but in this case there are no minutes, so it would not print a `0`,